src/components/MagicForm/MagicForm.js

```diff
@@ -1,107 +1,74 @@
-import {nanoid} from 'nanoid';
-import {useState} from 'react';
+//import {nanoid} from 'nanoid';
+//import {useState} from 'react';
 import styled from 'styled-components';
 
-export default function MagicForm({analyzeSpell}) {
-  const [savedOrders, SetSavedOrders] = useState([]);
+import {AriaOnlySpan, FrameSpan} from '../Stylesheet/StyledSpans.js';
 
+export default function MagicForm({saveSpellOrder, savedOrders}) {
   function handleSpell(event) {
     event.preventDefault();
     const Form = event.target;
```

Comment:
I would suggest to rename `Form` with lowercase letter `form`. Usually only components start with a capital letter.
```suggestion
    const form = event.target;
```